### ISSUE TITLE
exec UI page with terminal only - enabling embedded terminal in external UI

### ISSUE THREAD

**Issue by lchayoun** [hashicorp/nomad#7898]
Allow embedding the terminal of a specific task in external UI

**Comment by DingoEatingFuzz** [comment]
Hello @lchayoun, I'd love to hear more about your use case. What is it that makes you specifically want to embed the terminal in an external UI?

I ask that question strictly for my own curiosity. Unless I'm missing something, this is a duplicate of #7056. If the UI could be embedded as an iframe, then you could embed the exec page as an iframe wherever you like.

**Comment by lchayoun** [comment]
Hi @DingoEatingFuzz 
Reading the other issue, I think that this depends on #7056.
But iframe will not be enough because it will enable embedding the full page.
I'm interested of embedded the terminal only.

The use case is embedding it in a custom UI application that has other parts as well, without enabling switching between tasks (the left panel on the exec UI)

**Comment by DingoEatingFuzz** [comment]
Ah good point. I'll leave this issue open then.
